doc/articles/migrating-from-xamarin-to-net6.md

```diff
@@ -16,8 +16,8 @@ There are two ways to migrate an application to .NET 7.
 
    1. In a separate folder, create a new application from the templates using [`dotnet new unoapp -o MyApp`](get-started-dotnet-new.md) (MyApp being of the same name as your existing app), or by using the Visual Studio `Uno Platform App` template
```

Comment:
`[`dotnet new unoapp -o MyApp`](get-started-dotnet-new.md)`
@morning4coffe-dev can we use xref: instead of .md file here please